site/pages/what-we-do/header-slice/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styles from './style.css';
+import arrowImg from './arrow.png';
+
+const headerSlice = () => {
+  return (
+    <section className={styles.pageHeaderContainer}>
+      <div className={styles.left}>
+        <div className={styles.flexible}>
+          <h1 className={styles.pageHeader}>
+            <p>Do the</p>
+            <p>right thing</p>
+          </h1>
+          <img role="presentation" className={styles.arrowDown} src={arrowImg} />
```

Comment:
Using `role` here is good but having `alt=""` is actually better supported https://www.w3.org/WAI/tutorials/images/decorative/